src/main/java/com/nextstep/racingcar/Race.java

```diff
@@ -0,0 +1,43 @@
+package com.nextstep.racingcar;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Supplier;
+
+public class Race {
+    private List<Car> carList = new ArrayList<>();
+    private static final int THRESHOLD = 3;
+
+    public Race(int carCount, int moveLimit) {
+        for ( int ix = 0 ; ix < carCount ; ix ++ ) {
+            carList.add(new Car(moveLimit));
+        }
+    }
+
+    public List<Car> moveAndGet(Supplier<Integer> numberGenerator) {
+        for (Car car : carList) {
+            int number = numberGenerator.get();
+            tryMove(car, number);
+        }
+        return carList;
+    }
+
+    private void tryMove(Car car, int number) {
+        if (!isMove(number)) {
+            return;
+        }
+        car.move();
```

Comment:
```suggestion
        if (isMove(number)) {
            car.move();
        }
     
```

부정을 쓰는것보다 코드도 간결하고, 실수도 방지 할 수 있을 것 같아요.